pkg/deploy/config.go

```diff
@@ -378,6 +378,15 @@ func NewStagingConfig(ctx context.Context, stage string) *Config {
 			Account:   acc,
 			Dev:       &dev,
 		},
+		Providers: map[string]Provider{
+			"test-vault": {
+				Uses: "commonfate/testvault@v1",
+				With: map[string]string{
+					"apiUrl":   "https://prod.testvault.granted.run",
+					"uniqueId": "2BvQYmAfGQ82U9B60YlQVKD4BP2",
```

Comment:
```suggestion
					"uniqueId": ksuid.New().String(),
```

The `uniqueId` parameter will help us avoid collisions if multiple tests run at the same time.

Note: you'll need to update the imports in this file to add `ksuid`